types/blob-util/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for blob-util 1.2
+// Project: https://github.com/nolanlawson/blob-util#readme
+// Definitions by: Max Battcher <https://github.com/WorldMaker>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function createBlob(parts: any[], options?: { type: string }): Blob;
+export function createObjectURL(blob: Blob): string;
+export function revokeObjectURL(url: string): void;
+export function blobToBinaryString(blob: Blob): Promise<string>;
+export function binaryStringToBlob(binary: string, type?: string): Promise<Blob>;
+export function blobToBase64String(blob: Blob): Promise<string>;
+export function base64StringToBlob(base64: string, type?: string): Promise<Blob>;
+export function dataURLToBlob(dataURL: string): Promise<Blob>;
+export function imgSrcToDataURL(src: string, type?: string, crossOrigin?: string): Promise<string>;
+export function canvasToBlob(canvas: HTMLCanvasElement, type?: string): Promise<Blob>;
+export function imgSrcToBlob(src: string, type?: string, crossOrigin?: string): Promise<Blob>;
+export function arrayBufferToBlob(arrayBuff: ArrayBuffer, type?: string): Promise<Blob>;
+export function blobToArrayBuffer(blob: Blob): Promise<ArrayBuffer>;
+
+// TypeScript Version: 2.1
```

Comment:
If this is actually necessary, then add it at the end of the headers - but it isn't clear to me why it is for this library.